### PR TITLE
Test flathub bot with git ssh source

### PR DESCRIPTION
**Pull request by catsout**

### CODE REVIEW

File: generated-sources.json [hunk 1]
```diff
@@ -29482,13 +29482,13 @@
     },
     {
         "type": "git",
-        "url": "https://git@github.com/laurent22/node-emoji.git",
+        "url": "ssh://git@github.com/laurent22/node-emoji.git",
```
Comment (hfiguiere) [inline generated-sources.json:29485]:
```suggestion
        "url": "https://github.com/laurent22/node-emoji.git",
```

This is probably what you want.

File: generated-sources.json [hunk 2]
```diff
@@ -29482,13 +29482,13 @@
...
         "commit": "9fa01eac463e94dde1316ef8c53089eeef4973b5",
         "dest": "flatpak-node/git-packages/node-emoji-9fa01eac463e94dde1316ef8c53089eeef4973b5"
     },
     {
         "type": "git",
-        "url": "https://git@github.com/laurent22/uslug.git",
+        "url": "ssh://git@github.com/laurent22/uslug.git",
```
Comment (hfiguiere) [inline generated-sources.json:29491]:
```suggestion
        "url": "https://github.com/laurent22/uslug.git",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]
not sure what you wanted to achieve but for github these URL should work better

**Comment by catsout** [comment]
> not sure what you wanted to achieve but for github these URL should work better

Actually, the file changed is generated by `flatpak-builder-tools`.  
The tools generate the source with git ssh url.  
I manually change these to https: 
https://github.com/flathub/net.cozic.joplin_desktop/blob/99c2127bf6b819c0a4915ab884b4e272d40d5f49/update-sources.sh#L30b

**Comment by hfiguiere** [comment]
well, there is a bug in the tool.

**Comment by hfiguiere** [comment]
It should replace `//git@` by `//` as well on that line.

**Comment by catsout** [comment]
> It should replace //git@ by // as well on that line.

No, the tool generate some thing like `git+ssh://git@github.com/...`  
It just break the build of previous pr  
So I write a `sed` command to change urls to https, and I forgot to remove `git@`

**Comment by hfiguiere** [comment]
it shouldn't generate git+ssh: url either.

**Comment by catsout** [comment]
> it shouldn't generate git+ssh: url either.

@hfiguiere
https://github.com/flatpak/flatpak-builder-tools/pull/193

**Pull request closed without merging**